src/EditorFeatures/Test2/CodeFixes/CodeFixServiceTests.vb

```diff
@@ -321,6 +322,10 @@ Namespace Microsoft.CodeAnalysis.Editor.Implementation.CodeFixes.UnitTests
             Public Function IsGenerateDocumentationCommentOptionEnabledAsync() As Task(Of Boolean) Implements ICopilotOptionsService.IsGenerateDocumentationCommentOptionEnabledAsync
                 Return Task.FromResult(True)
             End Function
+
+            Public Function IsGenerateMethodImplementationOptionEnabledAsync() As Task(Of Boolean) Implements ICopilotOptionsService.IsGenerateMethodImplementationOptionEnabledAsync
```

Comment:
my preference would be naming *everything* consistently.  here you call it "GenerateMethodImplementation", but elsewhere you call it "ImplementNotImplementedExcception".